packages/commercetools/api-client/package.json

```diff
@@ -5,7 +5,7 @@
   "server": "server/index.js",
   "main": "lib/index.cjs.js",
   "module": "lib/index.es.js",
-  "types": "lib/index.d.ts",
+  "types": "lib/*/*/src/index.d.ts",
```

Comment:
I'm afraid that's not right. After the package is released, this path will be incorrect, and typings might stop working.

Reply:
Actually, we don't have `index.d.ts` inside `lib` folder

Reply:
We do when you build the package using `yarn build:ct:tools`